glean-core/src/storage/mod.rs

```diff
@@ -143,4 +143,41 @@ impl StorageManager {
 
         snapshot
     }
+
+    ///  Snapshot the experiments.
+    ///
+    /// ## Arguments:
+    ///
+    /// * `storage` - The database to get data from.
+    ///
+    /// ## Return value
+    ///
+    /// Returns a JSON representation of the stored data.
```

Comment:
This can be more explicit and say "experiment data" or something

Reply:
You're right. I've also documented the returned data structure here.